docs/index.rst

```diff
@@ -27,10 +27,10 @@ Table of Contents
    :caption: Contents:
 
    installation
-   data_input
+   work_at_beamlines
    summed_spectrum_fit
    data_output
-   work_at_beamlines
+   data_input
```

Comment:
Wouldn't it be logical to have the `data_input` before the `data_output`?

Reply:
no, because the goal is to de-emphasise 'make_hdf5' workflow.

Reply:
Oh, I see.